src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/TasksTest.kt

```diff
@@ -788,6 +788,254 @@ class TasksTest : IntegrationTestBase() {
         }
       }
     }
+
+    @Test
+    fun `Get tasks by taskType for a user returns the relevant placement requests tasks for a user page 2`() {
+      `Given a User`(roles = listOf(UserRole.CAS1_MATCHER)) { user, jwt ->
+        `Given a User` { otherUser, _ ->
+          `Given an Offender` { offenderDetails, inmateDetails ->
+            `Given a Placement Request`(
+              placementRequestAllocatedTo = user,
+              assessmentAllocatedTo = otherUser,
+              createdByUser = otherUser,
+              crn = offenderDetails.otherIds.crn,
+            )
```

Comment:
Might be neater to do this in a loop:

```
repeat(12) {
  `Given a Placement Request`(
    placementRequestAllocatedTo = user,
    assessmentAllocatedTo = otherUser,
    createdByUser = otherUser,
    crn = offenderDetails.otherIds.crn,
  )
}
```

Then you could also have that `12` set as a `var numberOfPlacementRequests = 12`, so you've got a connection between the number of placement requests you create and the result too.